### DISCUSSION TITLE
maybe a bug with this mod

### DISCUSSION THREAD

**Discussion by Mini-Jelly** [P3pp3rF1y/SophisticatedBackpacks#1035] [Q&A]
I am using version 1.19.4. In some cases, the b key cannot open the backpack, and the d key cannot move the character. I need to change the keys to make this mod run normally. After disabling this mod , everything is restored, and the d key becomes valid. I don't know if this is a dependency problem.

**Comment by P3pp3rF1y** [comment]
First of all I have no idea how you can actually run the mod in 1.19.4 as it was only made up to 1.19.2 in 1.19 major version. And from what you're describing it sounds like some mod messing with keybinds or you have conflicting keybinds. No idea how SB could be doing anything when D is pressed unless you have set one of the SB controls to use that key.

**Reply by Salandora** [reply]
I believe this is for the fabric version?
I have just discovered that I never updated the discussions link in the readme file of my fork...
